NAMESPACE

```diff
@@ -12,3 +12,5 @@ export(parse_url_path)
 export(route)
 export(route_link)
 export(router_ui)
+import(shiny)
```

Comment:
I realized based on http://r-pkgs.had.co.nz/description.html#dependencies that importing the whole package is bad practice. As a result, I closed #49. Please add `shiny::` reference everywhere shiny packages are used and remove importing shiny, and shiny::observeEvent from `create_router_callback` definition.

Reply:
I still can see importing the whole package in here.